habitat_sim/utils/profiling_utils.py

```diff
@@ -66,13 +66,12 @@ class RangeContext(ContextDecorator):
     statement. See also range_push.
     """
 
-    def __init__(self, msg: str):
+    def __init__(self, msg: str) -> None:
         self._msg = msg
 
-    def __enter__(self):
+    def __enter__(self) -> "RangeContext":
```

Comment:
Is this an example of "old Python 3.6 workaround of using the String type annotations which evaluate lazily"?

Reply:
Yep. In Python 3.7>= you just use `from __future__ import annotations` and it works automatically.